packages/kolibri-tools/build_kolibri_tools.js

```diff
@@ -30,3 +30,17 @@ kolibriPkg.dependencies = kolibriCorePkg.dependencies;
 fs.writeFileSync(kolibriPackageFile, JSON.stringify(kolibriPkg, undefined, 2), {
   encoding: 'utf-8',
 });
+
+/*
+ * Step 4: Copy the kolibri language_info.json into the kolibri-tools package for use externally
+ */
+
+const languageInfo = require(path.resolve(__dirname, '../../kolibri/locale/language_info.json'));
```

Comment:
This is the build tools - it is built inside the Kolibri repo, and makes a copy of this into the package, so that it is indeed independent of the Kolibri repo at publish time. Similarly to how we bundle up the core API spec to be accessible outside of the kolibri repo.